docs/README.md

```diff
@@ -17,8 +17,19 @@ For an introduction to this project, check out our [blog post][blog-post-link].
 
 ## Installation
 
+To install requirements for the DatasetBuilder class only:
+
+    pip install pan3d
+
+**Recommended**: to install requirements for the DatasetViewer user interface:
+
     pip install pan3d[viewer]
 
+**Optional**: to install requirements for Pangeo and ESGF catalogs, respectively:
+
+    pip install pan3d[pangeo]
+
+    pip install pan3d[esgf]
 
```

Comment:
Should we have a `pan3d[all]` option?

Reply:
Good idea, added in [86d19d1](https://github.com/Kitware/pan3d/pull/69/commits/86d19d129439582f084e4f4305591a5730b28d62)